component/tls.libsonnet

```
@@ -35,25 +36,73 @@ local secrets = std.filter(
   ]
 );
 
-local certs = std.filter(
-  function(it) it != null,
-  [
-    local cert = params.cert_manager_certs[c];
-    if cert != null then
+local kyvernoAnnotation = {
+  'syn.tools/openshift4-console': 'secret-target-namespace',
+};
+
+local makeCert(c, cert) =
+  if !std.member(inv.applications, 'kyverno') then
+    error 'You need to add component `kyverno` to the cluster to be able to deploy cert-manager Certificate resources for the the openshift web console.'
+  else
```

```suggestion
  assert std.member(inv.applications, 'kyverno') : 'You need to add component `kyverno` to the cluster to be able to deploy cert-manager Certificate resources for the the openshift web console.';
```

Would remove one indentation and make it easier to read IMO.

---

Adjusted in latest version.